Sources/PT.PM.Common/Nodes/UstUtils.cs

```diff
@@ -260,5 +259,29 @@ public static TextSpan GetTextSpan(this IEnumerable<Ust> usts)
 
             return 0;
         }
+
+        public static List<Expression> ExtractMultiChild(MultichildExpression multiChild)
```

Comment:
`multichild` because of you don't use upper case for `C` character in other places.